osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -77,7 +77,7 @@ static inline void flagsFromSysfs(const std::string& name, std::string& flags) {
       if (content[0] == '0' && content[1] == 'x') {
         unsigned long int lflags = 0;
         if (safeStrtoul(content.substr(2, content.size() - 3), 16, lflags)) {
-          flags = std::to_string(lflags);
+          flags |= lflags;
```

Comment:
Be specific here about the flags you want to get from sysfs.

Reply:
It actually getting all sysfs flags. Do you mean we want only IFF_PROMISC from sysfs?